backend/build.gradle.kts

```diff
@@ -57,6 +57,7 @@ dependencies {
     implementation("io.ktor:ktor-client-core:2.1.2")
     implementation("io.ktor:ktor-client-cio:2.1.2")
 
+    implementation("com.github.doyaaaaaken:kotlin-csv-jvm:1.7.0")
```

Comment:
jackson csv library messed up the columns, so i decided to use that one

Reply:
If jackson does not work properly, we could also try a more commonly used library by Apache Foundation:
https://commons.apache.org/proper/commons-csv/
I would not have trust issues for this lib.

Reply:
@maxammann i didn't read the columns in the correct order even i specified it in a data class. Tried for hours to get it work, maybe sth in the csv wasn't read properly but couldn't figure out what. The other library worked at instant. But i can try the apache one

Reply:
now apache commons csv is in use